### PR TITLE
[ETP-730] special characters not exporting

### PR DESCRIPTION
**Pull request by redscar**
### 🎫 Ticket

[ETP-730] 

### 🗒️ Description

HTML entities weren't being correctly decoded. I added a new line on 562 that decodes HTML entities. 

New line of code

Big fix

I have attached a list of HTML entities that you can copy/paste for attendance information to replicate the bug.
[List Of HTML Entities.ods](https://github.com/the-events-calendar/event-tickets/files/6650487/List.Of.HTML.Entities.ods)

### 🎥 Artifacts 

https://user-images.githubusercontent.com/12241059/121948996-f7371b80-cd25-11eb-9044-7d35e7a93a1c.mp4

### ✔️ Checklist
- [X] I've included a changelog entry both in readme.txt and changelog.txt files. 
- [X] My code is tested. 
- [X] My code has [proper inline documentation].

[ETP-730]: https://theeventscalendar.atlassian.net/browse/ETP-730

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 == Changelog ==
-
+* Fix - Exporting Attendees with HTML entities [ETP-730]
```
Comment (bordoni) [inline changelog.txt:2]:
```suggestion

* Fix - Prevent Attendees with HTML entities from importing broken [ETP-730]

```

**Review by bordoni — CHANGES_REQUESTED** [review]
Small changes.

File: src/Tribe/Attendees.php [hunk 2]
```diff
@@ -558,6 +558,8 @@ private function generate_filtered_list( $event_id ) {
 
 				// Handle custom columns that might have names containing HTML tags
 				$row[ $column_id ] = wp_strip_all_tags( $row[ $column_id ] );
+				//Decode HTML Entities
```
Comment (juanfra) [inline src/Tribe/Attendees.php:561]:
```suggestion
				// Decode HTML Entities.
```
There must be a space between the `//` and the comment. All comments need to have a stop at the end.

File: changelog.txt (hunk 1 above)
Comment (juanfra) [reply changelog.txt:2]: Please also add a stop before the ticket ID.

File: changelog.txt [hunk 3]
```diff
@@ -1,5 +1,5 @@
 == Changelog ==
-* Fix - Exporting Attendees with HTML entities [ETP-730]
+* Fix - Prevent Attendees with HTML entities from importing broken. [ETP-730]
```
Comment (juanfra) [inline changelog.txt:2]: This should be `exporting`, right?

**Pull request merged**